README.md

```diff
@@ -106,7 +107,7 @@ If you are a developer working on Scylla, please read the [developer guidelines]
 * The [users mailing list] and [Slack channel] are for users to discuss configuration, management, and operations of the ScyllaDB open source.
 * The [developers mailing list] is for developers and people interested in following the development of ScyllaDB to discuss technical topics.
 
-[Users mailing list]: https://groups.google.com/forum/#!forum/scylladb-users
+[Users' mailing list]: https://groups.google.com/forum/#!forum/scylladb-users
```

Comment:
I think (but I'm not sure, I'm not an MD expert) this needs to be the same as the  [users mailing list] link above. You can't change just one.

Reply:
you're right @nyh, if you look at resulting file with github ui, you'll notice the link is now broken.